dev/buildtool/halyard_commands.py

```diff
@@ -134,13 +134,19 @@ def publish_to_nightly(self, repository):
     env = dict(os.environ)
     env.update({
         'PUBLISH_HALYARD_BUCKET_BASE_URL': options.halyard_bucket_base_url,
-        'PUBLISH_HALYARD_DOCKER_IMAGE_BASE': options.halyard_docker_image_base
+        'PUBLISH_HALYARD_DOCKER_IMAGE_BASE': options.halyard_docker_image_base,
+        'PUBLISH_HALYARD_DOCKER_IMAGE_SRC_BASE': options.halyard_artifacts_docker_image_nightly_base,
+        'PUBLISH_HALYARD_DOCKER_IMAGE_TARGET_BASE': options.halyard_artifacts_docker_image_nightly_base,
```

Comment:
I think you left out the `_ARTIFACT` from these two?

Reply:
Gah. My bad - trying to do things too fast. Fixed.